ztp/du-profile-kube-compare/metadata.yaml

```diff
@@ -0,0 +1,197 @@
+# TODO: User may into diffs if running it against incorrect OCP version. CNF-13527
+Parts:
+  - name: required-cluster-logging
+    Components:
+      - name: cluster-logging
+        type: Required
+        requiredTemplates:
+          - path: required/cluster-logging/ClusterLogCatSource.yaml
```

Comment:
Not required. The RDS calls out only 1 catalog source (DefaultCatSource is the one we expect).

Reply:
removed